include/functions.h

```diff
@@ -23,6 +23,7 @@ f32 signF(f32 val);
 
 void* heap_malloc(s32 size);
 void* _heap_malloc(HeapNode* head, u32 size);
+u32 _heap_free(HeapNode* heapNodeList, void* addr_to_free);
```

Comment:
```suggestion
u32 _heap_free(HeapNode* heapNodeList, void* addrToFree);
```

Reply:
I should have known better :P

Reply:
all good! :)

Reply:
Is this resolved?

Reply:
Looks like it's still not renamed here

Reply:
Renamed, thank you for catching functions.h, I got the c file.